src/WebJobs.Script/ScriptHostBuilderExtensions.cs

```diff
@@ -157,12 +159,16 @@ public static IHostBuilder AddScriptHostCore(this IHostBuilder builder, ScriptAp
 
         public static void AddCommonServices(IServiceCollection services)
         {
+            // The scope for these services is beyond a single host instance.
```

Comment:
This applies when running in the WebHost, but `AddCommonServices` take effect within the JobHost scope.